alignak/objects/satellitelink.py

```diff
@@ -145,7 +145,7 @@ def create_connection(self):
         self.con = HTTPClient(address=self.arb_satmap['address'], port=self.arb_satmap['port'],
                               timeout=self.timeout, data_timeout=self.data_timeout,
                               use_ssl=self.use_ssl,
-                              strong_ssl=self.hard_ssl_name_check
+                              strong_ssl=self.hard_ssl_name_check, name='Arbiter'
```

Comment:
The arbiter is the destination here. It's kinda confusing to get the user agent for the destination and not the source.